docs/host/host.md

```diff
@@ -21,6 +21,28 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 ## Node Maintenance
 
+
+:::warning
+
+We found that there is a chance that VMs on a node can encounter an I/O error while the node enters maintenance mode. Please follow this procedure to avoid the problem before a fix is released:
```

Comment:
Meanwhile, how does a user recover from this? Would taking the node out of maintenance mode, add the taint, re-enter maintenance mode work?

Reply:
If a VM enters an I/O error, the only way to recover it is to stop it first and start it again. Let me add this.

Reply:
Updated in the issue description (somehow I think it's a better place).